services/director-v2/.env-devel

```diff
@@ -46,10 +46,13 @@ STORAGE_ENDPOINT=http://storage:8080
 
 SWARM_STACK_NAME=simcore
 
+# S3 configuration
 S3_ENDPOINT=172.17.0.1:9001
 S3_ACCESS_KEY=12345678
 S3_SECRET_KEY=12345678
 S3_BUCKET_NAME=simcore
+S3_SECURE=0
```

Comment:
you might add in the PR description what are the instructions for the Devops.
since you add S3_SECURE=0 here (which was already the case right?)

Reply:
`S3_SECURE=1` actually for osparc-master with CEPH. I can check the implications of this later if you want

Reply:
ah this is the .env-devel of the director-v2. my bad. I thought that was the general one.